src/main/java/minegame159/meteorclient/systems/modules/combat/Surround.java

```diff
@@ -80,10 +87,10 @@
             .build()
     );
 
-    private final Setting<Boolean> useEnderChests = sgGeneral.add(new BoolSetting.Builder()
-            .name("use-ender-chests")
-            .description("Will surround with ender chests if they are found in your hotbar.")
-            .defaultValue(false)
+    private final Setting<Block> block = sgGeneral.add(new EnumSetting.Builder<Block>()
```

Comment:
Wouldn't using `ItemListSetting` make more sense? Similar to the way it's used in [Trajectories](https://github.com/MeteorDevelopment/meteor-client/blob/dccdc2e8e154c2cecefe3bc8b4eb302eb191ea49/src/main/java/minegame159/meteorclient/systems/modules/render/Trajectories.java#L39)